lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -849,6 +849,27 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return llvm::Error::success();
   }
 
+  if (typeName == "Mod") {
+    NodeValue in;
+    ASSIGN_VALUE_OR_RETURN_ERR(in,
+                               getNodeValueOrCreateConstantByName(op.input(0)));
+    int64_t divisor;
+    ASSIGN_VALUE_OR_RETURN_ERR(divisor, loadInt(dict["divisor"]));
+
+    RETURN_ERR_IF_NOT(divisor >= 1, "Divisor must not be less than 1.");
+
+    bool signFollowDivisor = false;
+    if (dict.count("sign_follow_divisor")) {
+      ASSIGN_VALUE_OR_RETURN_ERR(signFollowDivisor,
+                                 loadInt(dict["sign_follow_divisor"]));
+    }
+
+    auto *node = G_.createModulo(opName, in, divisor, signFollowDivisor);
```

Comment:
At this point `divisor` is silently converted from 64-bit to 32-bit. Is it intentional? Caffe2 uses `int64_t`

Reply:
fixed